firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -791,7 +786,7 @@ static void process_ps_ctrl_brake_command(
 // 
 // *****************************************************
 static void process_psvc_chassis_state1(
-	const psvc_chassis_state1_data_s * const chassis_data )
+    const psvc_chassis_state1_data_s * const chassis_data )
 {
     // brake pressure as reported from the C-CAN bus
     int16_t brake_pressure = chassis_data->brake_pressure;
```

Comment:
Is this variable used anywhere but in the debug print statement?  Did something get lost during the update?  If not we can probably remove some of this code as it's actually dead code.